packages/datastore/datastore/src/storage/adapters/websql/filterToSQL.ts

```diff
@@ -0,0 +1,70 @@
+import invariant from "tiny-invariant";
+import { AllOperators, Filter } from "../../..";
+
+type OperatorToSQL = {
+    [P in keyof AllOperators]: (key: string, value: any) => string
+};
+
+const defaultOp = (op: string) => (
+    (key: string, value: any) => {
+        if ("string" === typeof value) {
+            value = `'${value}'`;
+        }
+        return `${key} ${op} ${value}`;
+    }
+);
+
+// TODO contains is tricky, verify solution
```

Comment:
Was this verified.

Reply:
I need to run this to verify it. We still have problem of running sample app